tests/test_gpu_common.py

```diff
@@ -475,8 +476,8 @@ def test_composite_buffering_tasking_multi_output(self):
         assert len([i for i in symbols if isinstance(i, Lock)]) == 2
         threads = [i for i in symbols if isinstance(i, PThreadArray)]
         assert len(threads) == 2
-        assert threads[0].size.data == 1
-        assert threads[1].size.data == 1
+        assert threads[0].size.size == 1
```

Comment:
size.size ?

Reply:
it's not a user-level Constant anymore. It's a Scalar, with a maxsize associated